domains/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -216,84 +216,50 @@ pub mod pallet {
-            sync_committee_period_update: SyncCommitteePeriodUpdateOf<T>,
+            light_client_update: LightClientUpdateOf,
         ) -> DispatchResult {
             let _sender = ensure_signed(origin)?;
 
             Self::check_bridge_blocked_state()?;
 
-            let sync_committee_period = sync_committee_period_update.sync_committee_period;
+            let sync_committee_period = light_client_update.sync_committee_period;
             log::info!(
                 target: "ethereum-beacon-client",
-                "💫 Received sync committee update for period {}. Applying update",
+                "💫 Received light client update for sync committee period {}. Applying update",
                 sync_committee_period
             );
 
-            if let Err(err) =
-                Self::process_sync_committee_period_update(sync_committee_period_update)
-            {
+            Self::process_light_client_update(light_client_update).map_err(|err| {
                 log::error!(
                     target: "ethereum-beacon-client",
-                    "💫 Sync committee period update failed with error {:?}",
+                    "💫 Light client update failed with error {:?}",
                     err
                 );
-                return Err(err);
-            }
+                err
+            })?;
 
             log::info!(
                 target: "ethereum-beacon-client",
-                "💫 Sync committee period update for period {} succeeded.",
+                "💫 Light client update for sync committee period {} succeeded.",
                 sync_committee_period
             );
 
             Ok(())
         }
 
-        #[pallet::weight(T::WeightInfo::import_finalized_header())]
-        #[transactional]
-        pub fn import_finalized_header(
-            origin: OriginFor<T>,
-            finalized_header_update: FinalizedHeaderUpdateOf<T>,
-        ) -> DispatchResult {
-            let _sender = ensure_signed(origin)?;
-
-            Self::check_bridge_blocked_state()?;
-
-            let slot = finalized_header_update.finalized_header.slot;
-
-            log::info!(
-                target: "ethereum-beacon-client",
-                "💫 Received finalized header for slot {}.",
-                slot
-            );
-
-            if let Err(err) = Self::process_finalized_header(finalized_header_update) {
-                log::error!(
-                    target: "ethereum-beacon-client",
-                    "💫 Finalized header update failed with error {:?}",
-                    err
-                );
-                return Err(err);
-            }
-
-            log::info!(
-                target: "ethereum-beacon-client",
-                "💫 Stored finalized beacon header at slot {}.",
-                slot
-            );
-
-            Ok(())
-        }
-
+        /// Import an execution header
         #[pallet::weight(T::WeightInfo::import_execution_header())]
         #[transactional]
```

Comment:
this is defualt. We dont need to have it explicitly IMO

Reply:
Make sense. will remove.